src/main/kotlin/no/nav/syfo/dialogmelding/apprec/consumer/ApprecConsumer.kt

```diff
@@ -28,7 +28,7 @@ class ApprecConsumer(
 
     suspend fun run() {
         try {
-            while (applicationState.ready && RECEIVED_APPREC_MESSAGE_COUNTER.count() < 1000) {
```

Comment:
&& RECEIVED_APPREC_MESSAGE_COUNTER.count() < 1000 har blitt liggende igjen ved en uteglemmelse - var ment for utprøving da dette ble satt i gang.